coral/src/app/features/topics/details/documentation/TopicDocumentation.tsx

```diff
@@ -112,12 +123,18 @@ function TopicDocumentation() {
   return (
     <>
       <PageHeader
-        title={"Documentation"}
+        title={"Readme"}
         primaryAction={{
-          text: "Edit documentation",
+          text: "Edit readme",
           onClick: () => setEditMode(true),
         }}
       />
+      <Box component={Typography.SmallText} marginBottom={"l2"}>
+        Readme provides essential information, guidelines, and explanations
```

Comment:
This is in both the pageHeader and here are both needed to be defined separately?

Reply:
Ha! Good catch! No, they don't.

Reply:
change: https://github.com/Aiven-Open/klaw/pull/1607/commits/e436d5b931be4d9edb529c20b67a5c44d7df198d